jobs/build/rhcos_sync/Jenkinsfile

```diff
@@ -207,6 +207,18 @@ node {
                 """)
             }
         }
+
+        // run publish_azure_marketplace when publishing RHCOS bootimages
+        stage("Publish azure marketplace") {
+            build(
+                job: 'build%2publish_azure_marketplace',
+                propagate: false,
```

Comment:
Any reason to have propagate as false?

Reply:
I thought it should not be a blocker for rhcos_sync, if rhcos sync step success and publish_azure_marketplace job failed, then the rhcos_sync job should success and we only need to take a look on marketplace job and fix and rebuild it.
anyway, I try to avoid rebuilding from rhcos_sync, but making rhcos_sync failed also works for me to raise alert.